modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -413,10 +569,10 @@ def _apply_list_of_funcs(call_queue, partition):  # pragma: no cover
     str
         The node IP address of the worker process.
     """
-    for func, f_args, f_kwargs in call_queue:
-        func = deserialize(func)
-        args = deserialize(f_args)
-        kwargs = deserialize(f_kwargs)
```

Comment:
I wonder if we are passing anything at all through this parameter that could lead to a hang? I believe objects created explicitly using `ray.put` should work correctly.

Reply:
> I wonder if we are passing anything at all through this parameter that could lead to a hang?

I think that we're indeed not passing anything like that via `func` nor via `f_kwargs`, but I thought that it might be a good idea to unify the approach across `f_args, funcs and f_kwargs` to avoid such problems in the future